### PR TITLE
TRUNK-5030 Collection.isEmpty() should be used to test for emptiness

### PR DESCRIPTION
**Pull request by reubenvarghese1**
## Description

Replaced all the other ways of checking if size is 0 by using isEmpty() function

## Related Issue

see https://issues.openmrs.org/browse/TRUNK-5030

## Checklist:

- [x] My pull request only contains one single commit.
- [x] My pull request is based on the latest master branch
  `git pull --rebase upstream master`.
- [x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.
- [x] My code follows the code style of this project.
- [x] I have read the **CONTRIBUTING** document.
- [x] All existing tests passed.

### CODE REVIEW

File: api/src/main/java/org/openmrs/api/db/hibernate/HibernateOrderDAO.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/**
+	/**
```
Comment (teleivo) [inline api/src/main/java/org/openmrs/api/db/hibernate/HibernateOrderDAO.java]: @reubenvarghese1 can you fix the formatting here please

**Comment by teleivo** [comment]
thanks @reubenvarghese1 👍

File: api/src/main/java/org/openmrs/api/db/hibernate/HibernateOrderDAO.java (hunk 1 above)
Comment (reubenvarghese1) [inline api/src/main/java/org/openmrs/api/db/hibernate/HibernateOrderDAO.java]: @teleivo I wonder why this test is failing now.I had tested the build before creating the PR.I'll look into it and the formatting :)

Comment (reubenvarghese1) [inline api/src/main/java/org/openmrs/api/db/hibernate/HibernateOrderDAO.java]: I just tried running that test in my PC and it compiled without any errors , now i'm kinda clueless regarding the error out here.Any ideas?

**Comment by teleivo** [comment]
@reubenvarghese1 please squash your two commits into one

**Comment by reubenvarghese1** [comment]
@teleivo Squashed.I think we are good to go :-)

**Comment by teleivo** [comment]
@reubenvarghese1 looks good to me, @dkayiwa :)

**Comment by dkayiwa** [comment]
Thanks @reubenvarghese1 for working on it and thanks @teleivo for reviewing! 😊

**Comment by teleivo** [comment]
thank you @reubenvarghese1 technical debt went down :)
see https://ci.openmrs.org/sonar/dashboard/index/1865?did=1

**Comment by reubenvarghese1** [comment]
Anytime @teleivo .I'm glad my contribution was of use :-)

**Pull request merged**